paging/paging-common/src/test/kotlin/androidx/paging/PagingDataDifferTest.kt

```diff
@@ -1577,10 +1886,14 @@ class PagingDataDifferTest(
                     ).also { pagingSources.add(it) }
                 }
             ),
+        pagingDataMapping: (PagingData<Int>) -> PagingData<Int> = { it },
```

Comment:
Since we never pass anything differently and it's a no-op anyway, let's just simplify and always do the wrapping for tracking here

Reply:
Done.